PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/platformPlugin.xml

```diff
@@ -122,11 +122,11 @@
     </group>
 
     <group id="AzureToolkit.AzureActionGroup"
-           class="com.intellij.openapi.actionSystem.DefaultActionGroup"
            text="Azure"
            description="Contains Azure tools plugin actions"
            icon="/icons/Azure.svg"
```

Comment:
@sdubov The icon renders as a "dot" in Rider right now, not sure what's happening there (but also should not be a blocker to ship eap01)